contracts/src/StakeTableV2.sol

```diff
@@ -889,6 +906,17 @@ contract StakeTableV2 is StakeTable, PausableUpgradeable, AccessControlUpgradeab
         revert DeprecatedFunction();
     }
 
+    /// @notice Set the minimum delegate amount
+    /// @param newMinDelegateAmount The new minimum delegate amount in wei
+    function setMinDelegateAmount(uint256 newMinDelegateAmount)
+        external
+        virtual
+        onlyRole(DEFAULT_ADMIN_ROLE)
+    {
+        minDelegateAmount = newMinDelegateAmount;
```

Comment:
didn't check if `newMinDelegateAmount` is > 0 in the event that we did want to set it to zero, we should discuss